### ISSUE TITLE
[TASK] Release 1.3.3

### ISSUE THREAD

**Issue by innobead** [longhorn/longhorn#5581]
**What's the task? Please describe.**
Action items for releasing v1.3.3

**Describe the items of the task (DoD, definition of done) you'd like.**
  - Pre-Release
    - [x] Regression test plan (manual) - @khushboo-rancher 
    - [x] Run e2e regression for pre-GA milestones (`install`, `upgrade`) - @yangchiu 
    - [x] Run security testing of container images for pre-GA milestones - @yangchiu 
    - [x] Verify longhorn chart PR to ensure all artifacts are ready for GA (`install`, `upgrade`)  @yangchiu  
       - https://github.com/longhorn/longhorn/tree/v1.4.1-rc2 (`longhorn/longhorn`)
       - https://github.com/longhorn/charts/tree/v1.4.x (`longhorn/charts`)
    - [x] Run core testing (install, upgrade) for GA build from the previous patch and the last patch of the previous feature release (1.4.0 & 1.3.2). - @yangchiu 
  - Release
    - [x] Release longhorn/chart from the release branch to publish to ArtifactHub
    - [x] Release note
	     - [x] Deprecation note
	     - [x] Upgrade notes including highlighted notes, deprecation, compatible changes, and others impacting the current users
  - Post-Release
    - [x] ~Create a new release branch of manager/ui/tests/engine/longhorn instance-manager/share-manager/backing-image-manager when creating the RC1~
    - [x] Update https://github.com/longhorn/longhorn/blob/master/deploy/upgrade_responder_server/chart-values.yaml @PhanLe1010 
	- [x] Add another request for the rancher charts for the next release (`1.4.2`) @rebeccazzzz  
  - Rancher charts: verify the chart is able to install & upgrade - @khushboo-rancher 
    - [x] rancher/image-mirrors update @weizhe0422 (@PhanLe1010 )
        - https://github.com/rancher/image-mirror/pull/377
    - [x] rancher/charts dev-2.6 & 2.7 branches for rancher marketplace @weizhe0422 (@PhanLe1010)
        - `dev-2.6`: https://github.com/rancher/charts/pull/2558
        - `dev-2.7`: https://github.com/rancher/charts/pull/2555

cc @longhorn/qa

**Comment by yangchiu** [comment]
- Verify longhorn chart PR to ensure all artifacts are ready for GA
    - pending by testing https://github.com/longhorn/longhorn/issues/5550

**Comment by yangchiu** [comment]
Verify longhorn chart PR to ensure all artifacts are ready for GA verified :

For both longhorn/longhorn v1.3.3-rc2 and longhorn/chart v1.3.3-rc2:

1. Fresh install - Pass
2. Upgrade from v1.3.2 to v1.3.3-rc2 - Pass
4. After install/upgrade to v1.3.3-rc2, below test items passed
   - Create a statefulset and write data
   - Upgrade to v1.3.3-rc2
   - Upgrade engine image
   - Scale down/up workload
   - Trigger replica rebuilding
   - Create system backup
   - Create volume backup and restored
   - Verify volume data 
5. Uninstall

**Comment by yangchiu** [comment]
Run e2e regression for pre-GA milestones (install, upgrade):

v1.3.3-rc2:
[amd64](https://ci.longhorn.io/job/public/job/v1.3.x/job/v1.3.x-longhorn-tests-sles-amd64/331/): 4 failures
[arm64](https://ci.longhorn.io/job/public/job/v1.3.x/job/v1.3.x-longhorn-tests-sles-arm64/286/): 2 failures
[amd64 upgrade](https://ci.longhorn.io/job/public/job/v1.3.x/job/v1.3.x-longhorn-upgrade-tests-sles-amd64/278/): 3 failures
[arm64 upgrade](https://ci.longhorn.io/job/public/job/v1.3.x/job/v1.3.x-longhorn-upgrade-tests-sles-arm64/275/): 2 failures

**Comment by yangchiu** [comment]
Run security testing of container images for pre-GA milestones:

[v1.3.3-rc2](https://ci.longhorn.io/job/private/job/image-security-scan/943/): 96 HIGH, 0 CRITICAL

**Comment by innobead** [comment]
>     - [ ] Run core testing (install, upgrade) for GA build from the previous patch and the last patch of the previous feature release 

@yangchiu Please help run the core test for the 1.3.3 GA candidate, since @roger-ryao is busy on one 1.3.3 regression testing. Thanks.

https://github.com/longhorn/longhorn/pull/5780
https://github.com/longhorn/charts/pull/109

cc @longhorn/qa

**Comment by yangchiu** [comment]
Run core testing (install, upgrade) for GA build:

[amd64](https://ci.longhorn.io/job/public/job/v1.3.x/job/v1.3.x-longhorn-upgrade-tests-sles-amd64/308/): 0 failure
[arm64](https://ci.longhorn.io/job/public/job/v1.3.x/job/v1.3.x-longhorn-tests-sles-arm64/316/): 0 failure
[amd64 upgrade](https://ci.longhorn.io/job/public/job/v1.3.x/job/v1.3.x-longhorn-upgrade-tests-sles-amd64/308/): 0 failure
[arm64 upgrade](https://ci.longhorn.io/job/public/job/v1.3.x/job/v1.3.x-longhorn-upgrade-tests-sles-arm64/303/): 0 failure

**Comment by innobead** [comment]
> * Post-Release
>   
>   * [ ]  ~Create a new release branch of manager/ui/tests/engine/longhorn instance-manager/share-manager/backing-image-manager when creating the RC1~
>   * [ ]  Update https://github.com/longhorn/longhorn/blob/master/deploy/upgrade_responder_server/chart-values.yaml @PhanLe1010
>   * [ ]  Add another request for the rancher charts for the next release (`1.4.2`) @rebeccazzzz
> * Rancher charts: verify the chart is able to install & upgrade - @khushboo-rancher
>   
>   * [ ]  rancher/image-mirrors update @weizhe0422 (@PhanLe1010 )
>     
>     * [docs(longhorn): Add longhorn 1.4.1 images rancher/image-mirror#360](https://github.com/rancher/image-mirror/pull/360)
>   * [ ]  rancher/charts dev-2.6 & 2.7 branches for rancher marketplace @weizhe0422 (@PhanLe1010)
>     
>     * `dev-2.6`: [Add Longhorn 1.4.1 chart with dev-2.6 rancher/charts#2478](https://github.com/rancher/charts/pull/2478)
>     * `dev-2.7`: [Add Longhorn 1.4.1 chart with dev-2.7 rancher/charts#2479](https://github.com/rancher/charts/pull/2479)
> 
> cc @longhorn/qa

@weizhe0422 @PhanLe1010 @rebeccazzzz Please help with post-release items. Thanks.

cc @longhorn/qa

**Comment by weizhe0422** [comment]
I've created PRs for Longhorn `v1.3.3` in Rancher `v2.6` and `v2.7`, and also tested to make sure it installs and starts correctly. In addition, the mirrored Longhorn images had been uploaded to the docker hub, and it is also verified that they can be pulled properly.

**Comment by innobead** [comment]
> I've created PRs for Longhorn `v1.3.3` in Rancher `v2.6` and `v2.7`, and also tested to make sure it installs and starts correctly. In addition, the mirrored Longhorn images had been uploaded to the docker hub, and it is also verified that they can be pulled properly.

@weizhe0422 @rebeccazzzz what's the status of these charts now? I know @khushboo-rancher have verified them, so are they merged already?

**Comment by weizhe0422** [comment]
@innobead Update current status: this three PRs includes mirror images and rancher chart files had been approved by @PhanLe1010, tested by @khushboo-rancher and got merged into `dev2-6`/`dev-2.7` by Rancher dev. 

But there is an UI issue([Issue#5794](https://github.com/longhorn/longhorn/issues/5794)) raised by @khushboo-rancher, and I will create PRs to fix it.

**Comment by innobead** [comment]
Thanks @weizhe0422 :+1:

**Comment by innobead** [comment]
>   * [ ]  Update https://github.com/longhorn/longhorn/blob/master/deploy/upgrade_responder_server/chart-values.yaml @PhanLe1010
>   * [ ]  Add another request for the rancher charts for the next release (`1.4.2`) @rebeccazzzz

@PhanLe1010 @rebeccazzzz can you check if these items are all done or not? If yes, mark the checkboxes.
